lib/flutter_sound.dart

```diff
@@ -83,11 +83,13 @@ class FlutterSound {
     }
   }
 
-  Future<String> startRecorder(String uri) async {
+  Future<String> startRecorder(String uri, {int sampleRate = 44100, int numChannels = 2}) async {
```

Comment:
I hope you to rename parameters like below unless you provide the same feature in `ios`.
Like
```
startRecorder(
  String uri, {  numChannels = 2, sampleRate = 44100 },
) {
...
}
```

Reply:
@JoniDS Sorry. I've just realized this `PR` support the feautre in #24. I will resolve this.